Engine/fupch.h

```diff
@@ -20,5 +20,5 @@
 #import <simd/simd.h>
 #endif
 #elif FUEGO_PLATFORM_WIN
-#include <windows.h>
+#include <windowsx.h>
```

Comment:
Why did you change the header?

Reply:
X version contains some useful methods for gettting X\Y position of your mouse